src/c_kzg_4844.c

```diff
@@ -1111,8 +1112,23 @@ static C_KZG_RET hash_to_bytes(uint8_t out[32],
   for (i = 0; i < n; i++)
     bytes_from_g1(&bytes[np + i * 48], &comms[i]);
 
-  hash(out, bytes, np + n * 48);
+  uint8_t* hash_output = calloc(32, sizeof(uint8_t));
```

Comment:
Out of curiosity why do you allocate hash_output on the heap? and not like: `uint8_t hash_output[32] = {0};`

apart from marginally better performance you also get less bookkeepping